lib/src/corepalette_to_colorscheme.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+import 'package:material_color_utilities/material_color_utilities.dart';
+
+extension CorePaletteToColorScheme on CorePalette {
```

Comment:
note: giving the extension a name is what makes it public, moved it to its own file for better organization